_posts/2021-01-13-hapi-fhir.md

```diff
@@ -0,0 +1,454 @@
+---
+layout: post
+authors: [youri_vermeir]
+title: 'Introduction to HAPI FHIR'
+image: /img/2021-01-23-hapi-fhir/hapi-logo.jpg
```

Comment:
You included hapi-logo.jpg, hapi-logo.png and hapi.png in the img directory. Please delete the images that are not used.